tests/utils.js

```diff
@@ -707,9 +707,9 @@ module.exports = {
   closeReloadModal: closeReloadModal,
 
   closeTour: async () => {
-    await element.all(by.css('.modal-dialog a.cancel')).each(async elm => {
-      await elm.click();
-    });
+    const closeButton = element(by.css('#tour-select a.btn.cancel'));
+    await browser.wait(() => closeButton.isPresent(), 5000);
```

Comment:
Why did you decide not to use the helper functions e.g. `waitElementToBeClickable` or `waitElementToPresentNative`?

Reply:
I think we'd have a circular dependency. The utils file doesn't depend on the helper file :)

Reply:
You're right. Totally missed that :)